Test Runner/UI Tests/RepresentedUITextViewTests.swift

```diff
@@ -12,7 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-@testable import ArcGISToolkit
 import SwiftUI
```

Comment:
As I mentioned, SwiftUI shouldn't be imported either. The only module that a UI test file should import is XCTest.

```suggestion
```